packages/components/molecules/f-searchbox/package.json

```diff
@@ -63,16 +63,16 @@
     "@justeat/browserslist-config-fozzie": ">=1.1.1",
     "@justeat/f-button": "3.x",
     "@justeat/f-error-message": "1.x",
-    "@justeat/f-mega-modal": "3.x",
+    "@justeat/f-mega-modal": "7.x",
```

Comment:
As mentioned, this version of Searchbox isn't being used in any consuming applications, but I've tested locally in Storybook with these new major versions and it seems to function fine!